contrib/cachetool.php

```diff
@@ -72,13 +72,13 @@
         $return = [$fullOptions];
     } elseif (count($options) > 0) {
         foreach ($options as $option) {
-            if ($option !== '') {
+            if (is_string($option) && $option !== '') {
                 $return[] = "--fcgi={$option}";
             }
         }
     }
 
-    return $return;
+    return $return ?: [''];
```

Comment:
I had to double check, but `[] ?: ['']` returns `['']`, so this makes sense and it should resolve the issue I'm seeing in 7.3.3.

Thank you! 👍